source/sdr.rst

```diff
@@ -181,19 +239,44 @@ The :math:`d` in :math:`dR_i` indicates a delta difference and :math:`p(i,k)` is
 
 Now we define the amount of sediment flux that is retained on any pixel in the flowpath using :math:`dR_i` as a weighted flow of upstream flux:
 
-.. math:: R_i=dR_i\cdot\left(\sum_{j\in\{pixels\ that\ drain\ to\ i\}}F_j p(i,j)+E'_i\right)
+.. math:: R_i=dR_i\cdot\left(\left(\sum_{j\in\{pixels\ that\ drain\ to\ i\}}F_j \cdot p(i,j)\right) + E'_i\right)
     :label: ri
 
 where :math:`F_j` is the amount of sediment-export-that-does-not-reach-the stream "flux", defined as:
 
-.. math:: F_i=(1-dR_i)\cdot\left(\sum_{j\in\{pixels\ that\ drain\ to\ i\}} F_j p(i,j)+E'_i\right)
+.. math:: F_i=(1-dR_i)\cdot\left(\left(\sum_{j\in\{pixels\ that\ drain\ to\ i\}} F_j \cdot p(i,j)\right) + E'_i\right)
     :label: fi
 
 Optional Drainage Layer
 ^^^^^^^^^^^^^^^^^^^^^^^
 
 In some situations, the index of connectivity defined by topography does not represent actual flow paths, which may be influenced by artificial connectivity instead. For example, sediments in urban areas or near roads are likely to be conveyed to the stream with little retention. The (optional) drainage raster identifies the pixels that are artificially connected to the stream, irrespective of their geographic position (e.g. their distance to the stream network). Pixels from the drainage layer are treated similarly to pixels of the stream network; in other words, the downstream flow path will stop at pixels of the drainage layer (and the corresponding sediment load will be added to the total sediment export).
 
+Defined Area of Outputs
+^^^^^^^^^^^^^^^^^^^^^^^
+
+SDR and several other model outputs are defined in terms of distance to stream (:math:`d_i`). Therefore, these outputs are only defined for pixels that drain to a stream on the map (and so are within the streams' watershed). Pixels that do not drain to any stream will have nodata in these outputs. Increasing the threshold flow accumulation value may shrink the defined area of these outputs, as fewer stream pixels exist and the watershed is smaller. Correspondingly, decreasing the threshold flow accumulation value may expand the defined area of these outputs.

```

Comment:
This is very confusing to me. I have never seen increasing the TFA shrink the output area, it just creates fewer tributaries. Increasing the TFA has never expanded the area, it just creates more tributaries. If it did "expand", it couldn't go past the boundary of the input data. I'm guessing that this verbiage is related to removing the pour points? If so, instead of the shrink/expand text that's here, I'd suggest further explaining why they might not drain to a stream (going off the map, or pits/errors in the DEM), and describing what the output will look like for pixels that do not drain to the stream (you'll see areas of NoData that can't be explained by missing data in your input rasters...) While I'm thinking about this, was it crazy to suggest that these pixels receive a distinctly bogus value like -9999, so they'd be really obvious to people, and they wouldn't have to pour through their inputs to see what's causing NoData in the output?

Reply:
Yes, this is new behavior related to removing the pour points. I tried to revise the text as you suggest and added an image comparing the stream and SDR outputs with different TFA values. I'm struggling to explain this effectively.

Having a separate "nodata" value like -9999 could definitely be useful, and I made a ticket for it: #621 However it would require more not-insignificant code changes. 3.9.1 is just a bugfix release and in the interest of getting it out, I think we should come back to that idea as a possible feature in 3.10 or so.

Reply:
Thanks for working on this @emlys!  I'm about to take off for the day, but will take a look at the graphics, and would be happy to help word the explanation.

And thanks for considering the value-other-than-NoData for disconnected pixels. No hurry, just an idea.

Reply:
Thanks @newtpatrol!